main.cpp

```diff
@@ -217,19 +249,30 @@ struct CarDealership
     int staff = 23;
     int carServiced = 40;
     int inventoryBought = 15;
+    CarDealership();
     float sellCars(float price);
     void hireSalesmen();
     int buyInventory(int inventory);
 };
 
+CarDealership::CarDealership()
+{
+    carInventory = 0;
+    carSold = 0;
+    staff = 0;
+    carServiced = 0;
+    inventoryBought = 0;
```

Comment:
This overwrites the default values that your variables have. 
remove these value assignments.
- [ ] revise